src/main/java/uk/gov/hmcts/juror/job/execution/service/contracts/SmtpService.java

```diff
@@ -1,8 +0,0 @@
-package uk.gov.hmcts.juror.job.execution.service.contracts;
-
-import uk.gov.hmcts.juror.job.execution.config.SmtpConfig;
-
-public interface SmtpService {
-
-    void sendEmail(SmtpConfig config, String subject, String text, String... recipients);
```

Comment:
We will want to keep this class for future use

Reply:
Dead code should not be in live code. If the code is required in the future, it can be retrieved from source control.